Unity3D/Assets/RosSharp/Scripts/ImagePublisher.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
```

Comment:
Please rename the variables as follows:
cameraToPublish -> camera
baseTopic -> topic
resWidth -> resolutionWidth
resHeight -> resolutionHeight
pub_id -> topicID or publicationID
rt -> renderTexture
viewport -> rect
screenshot -> texture2D